src/Ampersand/Classes/Relational.hs

```diff
@@ -14,23 +14,22 @@ class HasProps r where
   properties :: r -> AProps
 
 class Relational r where
-  isProp :: r -> Bool -- > tells whether the argument is a property
-  isImin :: r -> Bool -- > tells whether the argument is equivalent to I-
-  isTrue :: r -> Bool -- > tells whether the argument is equivalent to V
-  isFalse :: r -> Bool -- > tells whether the argument is equivalent to V-
-  isFunction :: r -> Bool
-  isTot :: r -> Bool --
-  isUni :: r -> Bool --
-  isSur :: r -> Bool --
-  isInj :: r -> Bool --
-  isRfx :: r -> Bool --
-  isIrf :: r -> Bool --
-  isTrn :: r -> Bool --
-  isSym :: r -> Bool --
-  isAsy :: r -> Bool --
-  isIdent :: r -> Bool -- > tells whether the argument is equivalent to I
-  isEpsilon :: r -> Bool -- > tells whether the argument is equivalent to I
-
+  isProp :: r -> Bool     -- > If True, then the argument is a property. Otherwise we don't know.
+  isIdent :: r -> Bool    -- > If True, then the argument is equivalent to I. Otherwise we don't know.
+  isImin :: r -> Bool     -- > If True, then the argument is equivalent to -I. Otherwise we don't know.
+  isTrue :: r -> Bool     -- > If True, then the argument is equivalent to V. Otherwise we don't know.
+  isFalse :: r -> Bool    -- > If True, then the argument is equivalent to -V. Otherwise we don't know.
+  isFunction :: r -> Bool -- > If True, then the argument is a total function. Otherwise we don't know.
+  isTot :: r -> Bool      -- > If True, then the argument is total. Otherwise we don't know.
+  isUni :: r -> Bool      -- > If True, then the argument is univalent. Otherwise we don't know.
+  isSur :: r -> Bool      -- > If True, then the argument is surjective. Otherwise we don't know.
+  isInj :: r -> Bool      -- > If True, then the argument is injective. Otherwise we don't know.
+  isRfx :: r -> Bool      -- > If True, then the argument is reflexive. Otherwise we don't know.
+  isIrf :: r -> Bool      -- > If True, then the argument is irreflexive. Otherwise we don't know.
+  isTrn :: r -> Bool      -- > If True, then the argument is transitive. Otherwise we don't know.
+  isSym :: r -> Bool      -- > If True, then the argument is symmetric. Otherwise we don't know.
+  isAsy :: r -> Bool      -- > If True, then the argument is antisymmetric. Otherwise we don't know.
```

Comment:
It would be better to not use Bool for this. A dedicated type would make more sence, and it will guareantee that `not` isn't used by accident. 
I am thinking something like 
~~~
data IsKnown = IsTrue | Unknown
~~~
On the other hand, I don't know if it is worth the hassle.